sdk/analytics/src/main/java/com/klaviyo/analytics/UserInfo.kt

```diff
@@ -1,11 +1,18 @@
 package com.klaviyo.analytics
 
+import com.klaviyo.analytics.DeviceProperties.backgroundData
```

Comment:
I don't love what this file is turning into. I had solutions in our very first SDK architecture designs that would separate state itself from important side-effects. I would like to fast follow this PR by implementing those designs. Roughly speaking that PR would move some of these changes to other places, but the overall logic would remain the same. Just better organized.